<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>472</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>FLIKA Docks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>File</string>
        </property>
        <widget class="QPushButton" name="openButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveAsButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save As</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>ROI Tools</string>
        </property>
        <widget class="QPushButton" name="plotAllButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Plot All</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
        <widget class="QPushButton" name="importROIButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Import ROIs</string>
         </property>
        </widget>
        <widget class="QPushButton" name="exportROIButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Export ROIs</string>
         </property>
        </widget>
        <widget class="QPushButton" name="point">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Point</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/point.png</normaloff>../images/point.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="freehand">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Freehand</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/freehand.png</normaloff>../images/freehand.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="line">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Line</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/line.png</normaloff>../images/line.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="rectangle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Rectangle</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/rect.png</normaloff>../images/rect.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave_Points"/>
    <addaction name="actionLoad_Points"/>
    <addaction name="actionLoad_ROI_File"/>
    <addaction name="actionSave_Movie"/>
    <addaction name="actionChange_Internal_Data_type"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuStacks">
     <property name="title">
      <string>Stacks</string>
     </property>
     <addaction name="actionSlice_Keeper"/>
     <addaction name="actionDeinterleave"/>
     <addaction name="actionZ_Project"/>
     <addaction name="actionPixel_Binning"/>
     <addaction name="actionFrame_Binning"/>
    </widget>
    <widget class="QMenu" name="menuWindow">
     <property name="title">
      <string>Overlay</string>
     </property>
     <addaction name="actionBackground"/>
     <addaction name="actionTime_Stamp"/>
     <addaction name="actionScale_Bar"/>
    </widget>
    <addaction name="menuStacks"/>
    <addaction name="actionMeasure"/>
    <addaction name="actionSet_value"/>
    <addaction name="menuWindow"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuMath">
     <property name="title">
      <string>Math</string>
     </property>
     <addaction name="actionMultiply"/>
     <addaction name="actionSubtract"/>
     <addaction name="actionPower"/>
     <addaction name="actionRatio"/>
     <addaction name="actionAbsolute_Value"/>
     <addaction name="actionSubtract_Trace"/>
    </widget>
    <widget class="QMenu" name="menuFilters">
     <property name="title">
      <string>Filters</string>
     </property>
     <addaction name="actionGaussian_Blur"/>
     <addaction name="separator"/>
     <addaction name="actionButterworth_Filter"/>
     <addaction name="actionMean_Filter"/>
     <addaction name="actionDifference_Filter"/>
     <addaction name="actionBoxcar_Differential"/>
     <addaction name="actionWavelet_Filter"/>
     <addaction name="actionFourier_Filter"/>
    </widget>
    <widget class="QMenu" name="menuBinary">
     <property name="title">
      <string>Binary</string>
     </property>
     <addaction name="actionThreshold"/>
     <addaction name="actionAdaptive_Threshold"/>
     <addaction name="actionCanny_Edge_Detector"/>
     <addaction name="separator"/>
     <addaction name="actionLogically_Combine"/>
     <addaction name="actionRemove_Small_Blobs"/>
     <addaction name="actionBinary_Dilation"/>
     <addaction name="actionBinary_Erosion"/>
    </widget>
    <addaction name="menuBinary"/>
    <addaction name="menuMath"/>
    <addaction name="menuFilters"/>
    <addaction name="actionImage_Calculator"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <widget class="QMenu" name="menuPuffs">
     <property name="title">
      <string>puffs</string>
     </property>
     <addaction name="actionFrame_by_frame_origin"/>
     <addaction name="actionAverage_origin"/>
     <addaction name="actionThreshold_cluster"/>
    </widget>
    <addaction name="menuPuffs"/>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="title">
     <string>Scripts</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDocs">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuProcess"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuScripts"/>
   <addaction name="menuDocs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDeinterleave">
   <property name="text">
    <string>Deinterleave</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
  </action>
  <action name="actionGaussian_Blur">
   <property name="text">
    <string>Gaussian Blur</string>
   </property>
  </action>
  <action name="actionButterworth_Filter">
   <property name="text">
    <string>Butterworth Filter</string>
   </property>
  </action>
  <action name="actionRatio">
   <property name="text">
    <string>Ratio by Baseline</string>
   </property>
  </action>
  <action name="actionBoxcar_Differential">
   <property name="text">
    <string>Boxcar Differential</string>
   </property>
  </action>
  <action name="actionThreshold">
   <property name="text">
    <string>Threshold</string>
   </property>
  </action>
  <action name="actionRemove_Small_Blobs">
   <property name="text">
    <string>Remove Small Blobs</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionMultiply">
   <property name="text">
    <string>Multiply</string>
   </property>
  </action>
  <action name="actionKymograph">
   <property name="text">
    <string>Kymograph</string>
   </property>
  </action>
  <action name="actionSlice_Keeper">
   <property name="text">
    <string>Slice Keeper</string>
   </property>
  </action>
  <action name="actionPython_Interpreter">
   <property name="text">
    <string>Python Interpreter</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="text">
    <string>Docs</string>
   </property>
  </action>
  <action name="actionLoad_ROI_File">
   <property name="text">
    <string>Load ROI File</string>
   </property>
  </action>
  <action name="actionWavelet_Filter">
   <property name="text">
    <string>Wavelet Filter</string>
   </property>
  </action>
  <action name="actionFrame_by_frame_origin">
   <property name="text">
    <string>Frame by frame origin</string>
   </property>
  </action>
  <action name="actionSet_Value">
   <property name="text">
    <string>Set Value</string>
   </property>
  </action>
  <action name="actionSet_value">
   <property name="text">
    <string>Set Value</string>
   </property>
  </action>
  <action name="actionAdaptive_Threshold">
   <property name="text">
    <string>Adaptive Threshold</string>
   </property>
  </action>
  <action name="actionLogically_Combine">
   <property name="text">
    <string>Logically Combine</string>
   </property>
  </action>
  <action name="actionDifference_Filter">
   <property name="text">
    <string>Difference Filter</string>
   </property>
  </action>
  <action name="actionCanny_Edge_Detector">
   <property name="text">
    <string>Canny Edge Detector</string>
   </property>
  </action>
  <action name="actionAbsolute_Value">
   <property name="text">
    <string>Absolute Value</string>
   </property>
  </action>
  <action name="actionZ_Project">
   <property name="text">
    <string>Z Project</string>
   </property>
  </action>
  <action name="actionBinary_Dilation">
   <property name="text">
    <string>Binary Dilation</string>
   </property>
  </action>
  <action name="actionBinary_Erosion">
   <property name="text">
    <string>Binary Erosion</string>
   </property>
  </action>
  <action name="actionImage_Calculator">
   <property name="text">
    <string>Image Calculator</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="text">
    <string>Measure</string>
   </property>
  </action>
  <action name="actionAverage_origin">
   <property name="text">
    <string>average_origin</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="text">
    <string>Background</string>
   </property>
  </action>
  <action name="actionTime_Stamp">
   <property name="text">
    <string>Time Stamp</string>
   </property>
  </action>
  <action name="actionScale_Bar">
   <property name="text">
    <string>Scale Bar</string>
   </property>
  </action>
  <action name="actionSave_Movie">
   <property name="text">
    <string>Save Movie</string>
   </property>
  </action>
  <action name="actionThreshold_cluster">
   <property name="text">
    <string>threshold_cluster</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="text">
    <string>Power</string>
   </property>
  </action>
  <action name="actionChange_Internal_Data_type">
   <property name="text">
    <string>Change Internal Data Type</string>
   </property>
  </action>
  <action name="actionPixel_Binning">
   <property name="text">
    <string>Pixel Binning</string>
   </property>
  </action>
  <action name="actionFourier_Filter">
   <property name="text">
    <string>Fourier Filter</string>
   </property>
  </action>
  <action name="actionFrame_Binning">
   <property name="text">
    <string>Frame Binning</string>
   </property>
  </action>
  <action name="actionSubtract_Trace">
   <property name="text">
    <string>Subtract Trace</string>
   </property>
  </action>
  <action name="actionSave_Points">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionLoad_Points">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionMean_Filter">
   <property name="text">
    <string>Mean Filter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>freehand</sender>
   <signal>pressed()</signal>
   <receiver>line</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
